ffv1.md

```diff
@@ -678,6 +680,10 @@ Note, this is different from JPEG-LS, which doesn’t use prediction in run mode
 
 # Bitstream
 
+An FFV1 bitstream is composed of a series of 1 or more `Frames` and (when required) a `Configuration Record`.
```

Comment:
I would invert `Configuration Record` and `Frames` here, because `Configuration Record` must be present first and it IMO looks a bit weird not to indicate it first in this sentence.